core/systems/input/managed_gamepad.gd

```diff
@@ -215,21 +215,32 @@ func grab() -> void:
 ## Processes all physical and virtual inputs for this controller. This should be
 ## called in a tight loop to process input events.
 func process_input() -> void:
+	# Only process input if we have a valid handle
+	if not phys_device or not phys_device.is_open():
+		return
+
 	# Calculate the amount of time that has passed since last invocation
 	var current_time := Time.get_ticks_usec()
 	var delta_us := current_time - _last_time
 	_last_time = current_time
 	var delta := delta_us / 1000000.0  # Convert to seconds
 
-	# Only process input if we have a valid handle
-	if not phys_device or not phys_device.is_open():
-		return
-
 	# Process all physical input events
 	var events = phys_device.get_events()
 	for event in events:
 		if not event or not event is InputDeviceEvent:
 			continue
+		
+		# TODO: Find a better way
```

Comment:
Why not use the same pattern as the rest of the handhelds and Mayo this to btn_base in handheld platform?

Reply:
Because Steam Deck/OpenSD works differently than the other handheld providers. There is no separate keyboard device that is available or created. It's just the gamepad.

OpenSD _does_ allow us to remap buttons to any other evdev input, but they will show up as available input events under the OpenSD gamepad device. If I configure the OpenSD device to map `BTN_BASE` to `BTN_BASE`, it screws up the Steam Input button order for the controller. If you map it to a keyboard key, though (like `KEY_F20`), then it does not screw up the button order.